bluepyemodel/efeatures_extraction/targets_configuration.py

```diff
@@ -296,7 +296,9 @@ def get_related_nexus_ids(self):
         uses = []
         for f in self.files:
             if f.id:
-                uses.append({"id": f.id, "type": "Trace"})
+                f_dict = {"id": f.id, "type": "Trace"}
+                if f_dict not in uses:
+                    uses.append({"id": f.id, "type": "Trace"})
```

Comment:
how about using f_dict here?

Reply:
Yes of course, good catch. done in next commit